### DISCUSSION TITLE
Problem installing qudi, qtconsole missing

### DISCUSSION THREAD

**Discussion by leonora-sewidan** [Ulm-IQO/qudi-core#129] [Q&A]
Hello, 

I used qudi for quite some time on my old PC and now wanted to install it on my new lab PC (windows 11). The installation of qudi-core went mostly fine, but when trying to run it from the cmd using 'qudi', I get the following error: 

ModuleNotFoundError: No module named 'qtconsole'
Unexpected return value 1. Exiting.

I tried installing  from PyPI and also installing from source (dev), but I get the same error each time. 

Is anyone familiar with this issue?

Edit: Never mind, I did something wrong in setting up the environments. Fixed it

**Comment by Neverhorst** [comment]
Glad to hear you found a solution quickly. Please feel free to ask similar questions anytime.
